test/e2e/api/matchMedia/map-bounding-box.html

```diff
@@ -0,0 +1,101 @@
+        }
+
+        mapml-viewer {
+            flex: 3;
+            height: 50vh;
+        }
+
+        .description {
+            padding: 20px;
+            flex: 1;
+        }
+
+        @media (max-width: 768px) {
+            body {
+                flex-direction: column;
+            }
+        }
+    </style>
+</head>
+
+<body>
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
+            const getBoundingBox = () => {
+                const map = document.querySelector('mapml-viewer');
+
+                // Retrieve the map extent
+                const extent = map.extent;
+
+                // Extract the PCRS values for the bounding box
+                const topLeftEasting = Math.trunc(extent.topLeft.pcrs.horizontal);
+                const topLeftNorthing = Math.trunc(extent.topLeft.pcrs.vertical);
+                const bottomRightEasting = Math.trunc(extent.bottomRight.pcrs.horizontal);
+                const bottomRightNorthing = Math.trunc(extent.bottomRight.pcrs.vertical);
+
+                // Format the media query string to detect overlap:
+                // (xminm < xmaxq) and (xmaxm > xminq) and (yminm < ymaxq) and (ymaxm > yminq)
+                const query = `(map-projection: OSMTILE) and (map-zoom < 14) and (map-top-left-easting < ${bottomRightEasting}) and (map-bottom-right-easting > ${topLeftEasting}) and (map-bottom-right-northing < ${topLeftNorthing}) and (map-top-left-northing > ${bottomRightNorthing})`;
+
+                let matcher = map.matchMedia(query);
+                const logResults = (e) => {
+                    if (e.target.matches) {
+                        layer.checked = true;
+                        layer.removeAttribute('hidden');
+                    } else {
+                        layer.checked = false;
+                        layer.hidden = true;
+                    }
+                };
+                matcher.addEventListener('change', logResults);
+
+                // create a layer to visually represent the query as the map moves
+                let f = `<map-layer checked label="test media query"><map-meta name="projection" content="OSMTILE"></map-meta>
+                    <map-meta name="cs" content="pcrs"></map-meta><map-feature><map-properties>${query}</map-properties>
+                    <map-geometry><map-polygon><map-coordinates>${topLeftEasting} ${topLeftNorthing}
+                    ${bottomRightEasting} ${topLeftNorthing} ${bottomRightEasting} ${bottomRightNorthing} ${topLeftEasting} ${bottomRightNorthing}
+                    ${topLeftEasting} ${topLeftNorthing}</map-coordinates</map-polygon></map-geometry></map-feature></map-layer>`;
+
+                const parser = new DOMParser();
+                const layer = parser
+                    .parseFromString(f, 'text/html')
+                    .querySelector('map-layer');
+                map.appendChild(layer);
+                return { matcher, logResults };
```

Comment:
Suggest to rename logResults to hideUnhideLayer or something along those lines. Function doesn't really need to return anything.